packages/stark-ui/src/modules/session-ui/entities/stark-session-ui-config.ts

```diff
@@ -0,0 +1,18 @@
+import { InjectionToken } from "@angular/core";
+
+/**
+ * The InjectionToken version of the config name
+ */
+export const STARK_SESSION_UI_CONFIG: InjectionToken<StarkSessionUiConfig> = new InjectionToken<StarkSessionUiConfig>(
+	"StarkSessionUiConfig"
+);
+
+/**
+ * Definition of the configuration object for the Stark Session UI module
+ */
+export class StarkSessionUiConfig {
+	/**
+	 * If the warning dialog effect is disabled or not
```

Comment:
Let's expand this description so that it is clear we use an effect for this. What about this?

"Whether the warning dialog should be displayed or not. If true, then the Ngrx Effects in charge of displaying the dialog won't be executed."

Reply:
much better